Header header
circle_detection/detection_results[] tracked_objects          # A list of detected objects from a single timestamp